packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/scope.types.ts

```diff
@@ -32,6 +32,9 @@ export interface TargetScope {
    */
   readonly domain: Range;
 
+  /** Whether this scope could expand contiguously to its siblings. */
+  readonly contiguous?: boolean;
+
```

Comment:
huh.  A bit strange to have this on the scope handler 🤔

Reply:
It's on the scope and not the handler itself.

Reply:
yeah I guess you're right 🤔